.github/workflows/linter.yml

```diff
@@ -0,0 +1,44 @@
+name: Lint Code Base
```

Comment:
Just wanted to make sure you meant to add this since it wasn't mentioned in PR

Reply:
That was my bad, forgot to sync my fork before submitting the pr. Should be fixed now